SU2_CFD/src/solvers/CEulerSolver.cpp

```diff
@@ -6407,27 +6412,15 @@ void CEulerSolver::BC_Giles(CGeometry *geometry, CSolver **solver_container, CNu
       Pressure_e /= config->GetPressure_Ref();
 
       /* --- Compute avg characteristic jump  --- */
-      if (nDim == 2){
-        c_avg[3] = -2.0*(AveragePressure[val_marker][iSpan]-Pressure_e);
-      }
-      else
-      {
-        c_avg[4] = -2.0*(AveragePressure[val_marker][iSpan]-Pressure_e);
-      }
+      c_avg[nDim + 1] = -2.0*(AveragePressure[val_marker][iSpan]-Pressure_e);
```

Comment:
```suggestion
      c_avg[nDim + 1] = 2.0*(Pressure_e-AveragePressure[val_marker][iSpan]);
```

Reply:
Is there a guideline regarding operations like this? I kept it this way as it is more consistent with the literature